test/utils/BaseTest.sol

```diff
@@ -7,30 +7,31 @@ import { IERC20 } from "../../contracts/src/interfaces/IERC20.sol";
 import { ERC20ForwarderFactory } from "../../contracts/src/token/ERC20ForwarderFactory.sol";
 
 contract BaseTest is Test {
-    address alice;
-    address bob;
-    address celine;
-    address dan;
-    address eve;
-
-    address minter;
-    address deployer;
-    address feeCollector;
-    address sweepCollector;
-    address governance;
-    address pauser;
-    address registrar;
-    address rewardSource;
+    address internal alice;
```

Comment:
why internal? also realize it would be a huge PITA but doesn't style guide say internal vars should have `_` prefix?

Reply:
There is a compiler warning for "visibility" should be specified. Yeah, the style guide does say that, but the style guide has not been applied consistently towards the Hyperdrive test suites. I don't think it's a bad idea to address this at some point, but this PR is not the time for that.